Assets/Scripts/GameObjects/Peppers/PepperPlant.cs

```diff
@@ -2,94 +2,126 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+// TODO: add option to disable wateringcan
 public class PepperPlant : InteractableObject
 {
     public override bool isInteractable { get { return true; }}
+
+    // TODO: i feel like most of this should be empty list because this creates an entity to drop on a pizza, right? It is not, itself, able to interact with the pizza.
```

Comment:
But isn't the droppable object that is spawned the pepper? So technically it's being controlled by the PepperPlant? I'm not sure

Reply:
honestly good question haha